internal/service/pipeline/pipeline/resource_pipeline.go

```diff
@@ -95,6 +96,12 @@ func ResourcePipeline() *schema.Resource {
 							Optional:    true,
 							Computed:    true,
 						},
+						"is_harness_code_repo": {
+							Description: "If the repo is harness code",
```

Comment:
nit: for consistency
```suggestion
							Description: "If the repo is harness code.",
```